uber/models.py

```diff
@@ -1317,19 +1317,20 @@ def change_badge(self, attendee, badge_type, badge_num=None):
             elif CUSTOM_BADGES_REALLY_ORDERED:
                 if badge_type in PREASSIGNED_BADGE_TYPES and old_badge_type not in PREASSIGNED_BADGE_TYPES:
                     return 'Custom badges have already been ordered; you can add new staffers by giving them an Attendee badge with a Volunteer Ribbon'
```

Comment:
This code block was being executed, but it should only apply pre-con since we don't want to automatically set the badge number to 0 during the event.